core/domain/interaction_registry_test.py

```diff
@@ -69,3 +72,16 @@ def test_get_all_specs(self):
 
         self.assertEqual(
             terminal_interactions_count, EXPECTED_TERMINAL_INTERACTIONS_COUNT)
+
+    def test_interaction_specs_json_sync_all_specs(self):
+        """Test to ensure that the interaction_specs.json file is upto date
+        with additions in the individual interaction files.
+        """
+        all_specs = interaction_registry.Registry.get_all_specs()
+
+        spec_file = os.path.join(
+            'extensions/interactions', 'interaction_specs.json')
```

Comment:
I think the / shouldn't be hardcoded? os.path.join I think is written to work cross-platform, and this would break on an OS which uses \ in the paths right (I think windows does this)?

I think you'll need to join 'extensions', 'interactions' and 'interaction_specs.json'

Reply:
Done